airflow/providers/teradata/example_dags/example_teradata_operator.py

```diff
@@ -0,0 +1,132 @@
+#
```

Comment:
same comment about examples

Reply:
system tests are defined under [system ](https://github.com/Teradata/airflow/tree/teradata_release_1.0.0/tests/system/providers/teradata) test directory. These example DAGs are defined with reference to [oracle](https://github.com/apache/airflow/tree/main/airflow/providers/oracle/example_dags). These example DAGs are refereed in [documentation](https://github.com/Teradata/airflow/blob/teradata_release_1.0.0/docs/apache-airflow-providers-teradata/operators/teradata_to_teradata.rst)

Reply:
All example dags will be converted to system tests. https://github.com/apache/airflow/issues/22447

Reply:
Got it. Do you want to remove example dags now or can be removed in coming release.

Reply:
example_dags = system_tests. The system tests have three purposes, that's why they should be combined:

* system tests runnable and exposed via test dashboard
* example dags that people can copy and start with directly from GitHub
* the excerpts of the examples are embedded in Howto documentation

This way there is only one place where those examples/system tests are defined and there is much higher probability they will be kept in a good state when things change because for example mypy / ruff/ linting will check them for correctness and hipefully system test dashboard will keep them running in Green state

That's why all of those should be merged and kept in `tests/system` and they should be referred to in the howto documentation - you can take a look at google provider or amazon provider for example to see how it is done.

Reply:
The problem with having SEPARATE example_dags is that they have very high chance to be out-dated. This happens very, very quickly.

Reply:
okay thank you. Let me merge these and will keep only in tests/system.

Reply:
Removed example_dags and updated references with tests/system.